app/graphql/resolvers/billable_metrics_resolver.rb

```diff
@@ -12,14 +12,17 @@ class BillableMetricsResolver < GraphQL::Schema::Resolver
     argument :limit, Integer, required: false
     argument :search_term, String, required: false
 
+    argument :aggregation_types, [Types::BillableMetrics::AggregationTypeEnum], required: false
+
     type Types::BillableMetrics::Object.collection_type, null: false
 
-    def resolve(ids: nil, page: nil, limit: nil, search_term: nil)
+    def resolve(ids: nil, page: nil, limit: nil, search_term: nil, aggregation_types: nil)
       validate_organization!
 
       query = ::BillableMetricsQuery.new(organization: current_organization)
       result = query.call(
         search_term:,
+        aggregation_types:,
```

Comment:
Should we move it in `filters` field? Wdyt?

Reply:
yeah tbh I was not sure what to put into filter or not, but yeah it makes more sense in a way ! Will do